ee/session_recordings/playlist_counters/recordings_that_match_playlist_filters.py

```diff
@@ -458,14 +490,21 @@ def count_recordings_that_match_playlist_filters(playlist_id: int) -> None:
 
 
 def enqueue_recordings_that_match_playlist_filters() -> None:
-    all_playlists = (
-        SessionRecordingPlaylist.objects.filter(
-            deleted=False,
-            filters__isnull=False,
-        )
-        .filter(Q(last_counted_at__isnull=True) | Q(last_counted_at__lt=timezone.now() - timedelta(hours=2)))
-        .order_by(F("last_counted_at").asc(nulls_first=True))[: settings.PLAYLIST_COUNTER_PROCESSING_PLAYLISTS_LIMIT]
-    )
+    base_query = SessionRecordingPlaylist.objects.filter(
+        deleted=False,
+        filters__isnull=False,
+    ).filter(Q(last_counted_at__isnull=True) | Q(last_counted_at__lt=timezone.now() - timedelta(hours=2)))
+
+    total_playlists_count = base_query.count()
+    all_playlists = base_query.order_by(F("last_counted_at").asc(nulls_first=True))[
```

Comment:
and then the slice here does load all of the models, but only up to the count limit

all of which means we can count the total playlists in the database to process, without needing to load them all